poster/poster.tex

```diff
@@ -0,0 +1,428 @@
+
+\end{column} % End of column 2.2
+
+\end{columns} % End of the split of column 2
+
+\end{column} % End of the second column
+
+\begin{column}{\sepwid}\end{column} % Empty spacer column
+
+\begin{column}{\onecolwid} % The third column
+
+%----------------------------------------------------------------------------------------
+%	CONCLUSION
+%----------------------------------------------------------------------------------------
+
+\begin{block}{Conclusion}
+The review concludes that fuel cycle simulation tools approach scenario objective 
+functions in various ways.
+\begin{itemize}
+	\item wrap in external optimizer
+	\item look-ahead methods to predict input validity
+\end{itemize} 
+These methods fail to realistically model the process by which utilities, governments,
+and other stakeholders actually make facility deployment decisions.
+Dynamic, demand-driven facility deployment may be enabled by algorithms in use in
+other fields. Different deployment models differ in terms of both compute
+speed, flexibility (in terms of the range of scenarios capably
+simulated), and robustness (in terms of consistent fidelity of the modeling results).
+Finally, current NFC simulators may more flexibly support demand-driven deployment
+through incorporation of non-optimizing algorithms such as ARMA \cite{woodard_stationarity_2011} and ARCH \cite{li_kernal_2016},
+deterministically optimizing methods such as those collected in GCAM \cite{edmonds_advanced_1994} and
+MARKAL \cite{fishbone_markal_1981}, or stochastic optimization techniques such as Markov Switching Models \cite{ansari_predicting_2015}.
+\end{block}
+
+%----------------------------------------------------------------------------------------
+%	ACKNOWLEDGEMENTS
+%----------------------------------------------------------------------------------------
+
+\setbeamercolor{block title}{fg=red,bg=white} % Change the block title color
+
+\begin{block}{Acknowledgements}
+
+This research is being performed using funding received
+from the DOE Office of Nuclear Energy's Nuclear Energy
+University Programs under award number 16-10512.
+
+\end{block}
+
+
+%----------------------------------------------------------------------------------------
+%	CONTACT INFORMATION
+%----------------------------------------------------------------------------------------
+
+\setbeamercolor{block alerted title}{fg=black,bg=norange} % Change the alert block title colors
+\setbeamercolor{block alerted body}{fg=black,bg=white} % Change the alert block body colors
+
+\begin{alertblock}{Contact Information}
+
+\begin{itemize}
+	
+	\item Web: \href{arfc.github.io}{arfc.github.io}
+	\item Email: \href{mailto:jbae11@illinois.edu}{jbae11@illinois.edu}
+	\item Phone: +1 (217) 377-5784
+\end{itemize}
+
+\end{alertblock}
+
+\begin{center}
+\begin{tabular}{ccc}
+\includegraphics[width=0.4\linewidth]{logo.png} & \hfill & \includegraphics[width=0.4\linewidth]{neup.png}
```

Comment:
we should also include the ergs logo.

![http://www.ergs.sc.edu/static/logo-banner-text-upper.png](http://www.ergs.sc.edu/static/logo-banner-text-upper.png)

or 

![http://www.ergs.sc.edu/static/logo-curves.svg](http://www.ergs.sc.edu/static/logo-curves.svg)